order/event-handler/src/main/java/org/example/event/sourcing/order/poc/handler/order/config/TopicConfig.java

```diff
@@ -0,0 +1,21 @@
+package org.example.event.sourcing.order.poc.handler.order.config;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.config.TopicBuilder;
+
+import static org.example.event.sourcing.order.poc.common.model.event.OrderEvent.ORDER_TOPIC;
+import static org.example.event.sourcing.order.poc.common.model.event.OrderEvent.ORDER_TOPIC_PARTITION;
+
+@Configuration
```

Comment:
same as [above](https://github.com/NoahHsu/event-sourcing-order-poc/pull/34/files#r1070234550)

Reply:
revised in [df96d37](https://github.com/NoahHsu/event-sourcing-order-poc/pull/34/commits/df96d379a07fcea22fb985477b5766b3d6bf6ad1)